docs/testing/releases/740.md

```diff
@@ -0,0 +1,86 @@
+3. Verify the button is centered by default.
+4. Verify you can change the alignment of the button using the _Justify items_ dropdown inherited from the _Buttons_ block.
+![imatge](https://user-images.githubusercontent.com/3616980/160625173-f9ad42ed-b769-42e3-9ad8-3f3abe60b61c.png)
+5. Test again with a classic theme (ie: Storefront).
+
+### Remove the ToggleButtonControl in favor of ToggleGroupControl. ([5967](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5967))
+
+1. Check out this branch and compile the changes
+2. Activate a **block theme**, like Twenty Twenty Two
+3. Create a new page, and add the following blocks:
+   - Active Filters
+   - Attribute Filter
+   - Price Filter
+   - Product Categories
+   - Reviews
+4. Ensure that options are using the new component and are working correctly
+
+## Feature Plugin
+
+### The order summary area for the Cart and Checkout Blocks is now powered by Inner Blocks allowing for more customizations and extensibility. ([6065](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6065))
+
+1. Go to a page and add the Checkout block.
+2. Go to a page and add the Cart block
+3. In the editor, inside each block, select the Order summary and notice that you can select the following inner blocks:
+    | Cart | Checkout |
+    | --------------- | --------------- |
+    |  Heading, Subtotal, Cart Items, Fees, Coupon Form, Shipping, Taxes | Subtotal, Cart Items, Fees, Coupon Form, Shipping, Taxes |
+4. Try to remove or move some of the inner blocks and save.
+5. Check on the website that your changes are reflected. Note that Taxes, Fees and Coupons will appear on the website only if the shop supports them.
+6. For the Cart > Shipping inner block make sure you can enable disable the shipping calculator and that the change is visibile in the website
+7. Enable Taxes on your website (`/wp-admin/admin.php?page=wc-settings`) by clicking  `Enable tax rates and calculations`
+8. Go to the C & C blocks and select the Taxes inner block. 
+9. In the inner block's setting toggle on / of `Show rate after tax name` and make sure that is reflected in the website
+10. Try to place an order and make sure it's successful.
+
+### Increase Cart product quantity limit. ([6202](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6202))
+
+1. Go to the shop page and add any product to the cart
+2. Go to the Cart block page and increase product quantity to 9999
+
+### Decode HTML entities when formatting Store API error messages. ([5870](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5870))
+
+1. Add a callback to the wooocommerce_store_api_validate_cart_item action and throw a RouteException that includes HTML entities.
+2. Open the Cart Block.
+3. You should see a notice with the message you added to your Exception.
+4. Go to the Checkout Block.
+5. Attempt to place the order.
+6. You should see the same message, with entities decoded.
```

Comment:
These user-facing instructions require code editing to add a callback, can we change them so that users without the ability to do this can test them? cc @manospsyx and @senadir

Reply:
Testing notes removed. They can't be tested as a used!